### PR TITLE
[4.0] TinyMCE configuration option

### PR DESCRIPTION
**Pull request by brianteeman**
Due to a change between tinyMCE 4 and 5 although you could see and use the text colour and text background colour buttons in the toolbar you could not see them in the plugin so you lost the ability to add/remove those buttons to different sets

requires npm -i

Go to the plugin
check that Set 0 shows the two buttons
check that the collection of available buttons shows the two buttons

PS That took way too long to work out where the problem was :(

Pull Request for Issue #26333

### CODE REVIEW

**Comment by brianteeman** [comment]
Any help on fixing hound would be appreciated - it all looks fine to me

**Comment by richard67** [comment]
Just remove the single quotes around the word “forecolor”

File: build/media_source/plg_editors_tinymce/js/tinymce-builder.es6.js [hunk 1]
```diff
@@ -60,11 +61,11 @@ Joomla = window.Joomla || {};
...
       'flip-horizontally': '<svg width="24" height="24"><path d="M14 19h2v-2h-2v2zm4-8h2V9h-2v2zM4 7v10c0 1.1.9 2 2 2h3v-2H6V7h3V5H6a2 2 0 0 0-2 2zm14-2v2h2a2 2 0 0 0-2-2zm-7 16h2V3h-2v18zm7-6h2v-2h-2v2zm-4-8h2V5h-2v2zm4 12a2 2 0 0 0 2-2h-2v2z" fill-rule="nonzero"/></svg>',
       'flip-vertically': '<svg width="24" height="24"><path d="M5 14v2h2v-2H5zm8 4v2h2v-2h-2zm4-14H7a2 2 0 0 0-2 2v3h2V6h10v3h2V6a2 2 0 0 0-2-2zm2 14h-2v2a2 2 0 0 0 2-2zM3 11v2h18v-2H3zm6 7v2h2v-2H9zm8-4v2h2v-2h-2zM5 18c0 1.1.9 2 2 2v-2H5z" fill-rule="nonzero"/></svg>',
+      'forecolor': '<svg width="24" height="24"><g fill-rule="evenodd"><path id="tox-icon-text-color__color" d="M3 18h18v3H3z"/><path d="M8.7 16h-.8a.5.5 0 0 1-.5-.6l2.7-9c.1-.3.3-.4.5-.4h2.8c.2 0 .4.1.5.4l2.7 9a.5.5 0 0 1-.5.6h-.8a.5.5 0 0 1-.4-.4l-.7-2.2c0-.3-.3-.4-.5-.4h-3.4c-.2 0-.4.1-.5.4l-.7 2.2c0 .3-.2.4-.4.4zm2.6-7.6l-.6 2a.5.5 0 0 0 .5.6h1.6a.5.5 0 0 0 .5-.6l-.6-2c0-.3-.3-.4-.5-.4h-.4c-.2 0-.4.1-.5.4z"/></g></svg>',
```
[the remark this answers is not in the document]
Comment (Fedik) [reply build/media_source/plg_editors_tinymce/js/tinymce-builder.es6.js]: try change `'forecolor':` to `forecolor:`

File: build/media_source/plg_editors_tinymce/js/tinymce-builder.es6.js [hunk 2]
```diff
@@ -25,6 +25,7 @@ Joomla = window.Joomla || {};
       'align-right': '<svg width="24" height="24"><path d="M5 5h14c.6 0 1 .4 1 1s-.4 1-1 1H5a1 1 0 1 1 0-2zm6 4h8c.6 0 1 .4 1 1s-.4 1-1 1h-8a1 1 0 0 1 0-2zm0 8h8c.6 0 1 .4 1 1s-.4 1-1 1h-8a1 1 0 0 1 0-2zm-6-4h14c.6 0 1 .4 1 1s-.4 1-1 1H5a1 1 0 0 1 0-2z" fill-rule="evenodd"/></svg>',
       'arrow-left': '<svg width="24" height="24"><path d="M5.6 13l12 6a1 1 0 0 0 1.4-1V6a1 1 0 0 0-1.4-.9l-12 6a1 1 0 0 0 0 1.8z" fill-rule="evenodd"/></svg>',
       'arrow-right': '<svg width="24" height="24"><path d="M18.5 13l-12 6A1 1 0 0 1 5 18V6a1 1 0 0 1 1.4-.9l12 6a1 1 0 0 1 0 1.8z" fill-rule="evenodd"/></svg>',
+      'backcolor': '<svg width="24" height="24"><g fill-rule="evenodd"><path id="tox-icon-highlight-bg-color__color" d="M3 18h18v3H3z"/><path fill-rule="nonzero" d="M7.7 16.7H3l3.3-3.3-.7-.8L10.2 8l4 4.1-4 4.2c-.2.2-.6.2-.8 0l-.6-.7-1.1 1.1zm5-7.5L11 7.4l3-2.9a2 2 0 0 1 2.6 0L18 6c.7.7.7 2 0 2.7l-2.9 2.9-1.8-1.8-.5-.6"/></g></svg>',
```
[the remark this answers is not in the document]
Comment (Fedik) [reply build/media_source/plg_editors_tinymce/js/tinymce-builder.es6.js]: try change `'backcolor':` to `backcolor:`

**Comment by brianteeman** [comment]
Thanks guys - I misread the message

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on b7b4c4bde45f2d4e80a4e0454360259f21f536a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27465">issues.joomla.org/tracker/joomla-cms/27465</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b7b4c4bde45f2d4e80a4e0454360259f21f536a6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27465">issues.joomla.org/tracker/joomla-cms/27465</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27465">issues.joomla.org/tracker/joomla-cms/27465</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**